1-exercises/C-while-loop-with-array/exercise.js

```diff
@@ -17,7 +17,13 @@ const BIRTHDAYS = [
 ];
 
 function findFirstJulyBDay(birthdays) {
-    // TODO
+    let i = 0;
+
+    for (i = 0; ; i++)
```

Comment:
Nice attempt with a for loop, but the  exercise calls for a while loop to be used. How would we do this? Remember, a while loop is used to say "While something is true, do the following".

Reply:
Hint: try using a while loop where the while loop increments the "i" variable as long as arr[i]!== "July". After the while loop block, the "i"variable should reference the July entry.